src/common/containers/build-details.js

```diff
@@ -80,10 +78,15 @@ class BuildDetails extends Component {
             </div>
           </div>
         }
-        <HelpInstallSnap
-          headline='To test the latest successful build on your PC or cloud instance:'
-          name='foo'
-        />
+        { isFetching &&
+          <div className={styles.spinner}><Spinner /></div>
+        }
+        { snap &&
+          <HelpInstallSnap
+            headline='To test the latest successful build on your PC or cloud instance:'
+            name={ snap.store_name }
```

Comment:
It's called `store_name` in LP, and makes sense in that context (also hard to change now); BSI is just consuming the existing API here.